data/roles.js

```diff
@@ -186,299 +193,311 @@ const roleData = {
             ],
         },
         "Mimic": {
             alignment: "Village",
             description: [
-                "Can choose to mimic another player at night and will change based on their alignment.",
-                "Will steal role if target was town, become villager if target was mafia, and become amnesiac otherwise.",
+                "Chooses a player at night and attempt to mimic their role.",
+                "If player is town, mimic steals their role and that player becomes a villager.",
+                "If player is mafia, mimic becomes villager.",
+                "If player is independent or monster, mimic becomes amnesiac."
             ],
         },
         "Judge": {
             alignment: "Village",
             description: [
-                "Votes weight is 2."
+                "Vote weight is worth 2 players votes in day meeting."
             ],
         },
         "Party Host": {
             alignment: "Village",
             description: [
-                "Can choose to host a party for everyone to attend once per game.",
+                "Chooses to host a party during day meeting for everyone to attend once per game on the following night.",
+                "Everyone will share a night meeting."
             ],
         },
         "Loudmouth": {
             alignment: "Village",
             description: [
-                "Will cry out and reveal who visited them.",
-                "Sees self as blue."
+                "If visited, cries out the idenity of players who visited them during the night.",
+                "Appears as villager to self."
             ],
         },      
         "Ghost": {
             alignment: "Village",
             description: [
-                "Appears as villager until death.",
-                "Once dead, may visit one person a night and roleblock them.",
+                "Appears as villager to self, until dead.",
+                "Once dead, chooses to block one player each night."
             ],
             graveyardParticipation: "self",
         },
         "Lightkeeper": {
             alignment: "Village",
             description: [
-                "Causes an eclipse during the day following their death.",
-                "All votes and speech are anonymous during an eclipse."
+                "Following their death, causes an eclipse during the day",
+                "During an eclipse all speech and votes are anonymous."
             ],
         },
         "Resurrectionist": {
             alignment: "Village",
             description: [
-                "Once per game, visits one dead person during the night.",
-                "That person will be resurrected.",
+                "Visits a dead player during the night once per game.",
+                "That player will be resurrected the following day.",
+                "If players identity was revealed upon death, they will remain revealed when ressurected."
```

Comment:
Grammar. Should be "player's" instead of "players."
Spelling. Should be "resurrected" instead of "ressurected."